packages/kbn-search-index-documents/components/result/result_metadata.ts

```diff
@@ -44,10 +44,36 @@ export const resultMetaData = (result: SearchHit): MetaDataProps => ({
   score: result._score,
 });
 
+const MAPPING_TYPE_ORDER = ['semantic_text', 'dense_vector', 'sparse_vector'];
+const SPECIAL_NAME_FIELDS = ['headings'];
+
+export const reorderFieldsInImportance = (fields: FieldProps[]) => {
```

Comment:
Since this is exported a comment about what this does would be good for the public API